### DISCUSSION TITLE
Missed some things with MKT update

### DISCUSSION THREAD

**Discussion by MusicIsTheBest** [MKT-Playbook/Hub#4] [General]
1. The points cap values go higher now with quad uncapped.

2. The driver, Yoshi Mii Racing Suit, is missing correct coverage for the favorite track, RMX Donut Plains 1.

3.  Slow to calculate tours, cards where it'll display a value and then calculate more and then the final value. Makes for a jittery experience.

**Comment by TitusPosiadly** [comment]
Hey there, thanks for the requests. I tried to address each point individually down below. Let me know if you still have any questions or concerns.

1. Yes, I already implemented the changes regarding the new point cap in version 1.2.0 (currently in development) but I am still implementing the changes regarding the spotlight shop and the calculation for battle courses. It will take me a couple more days to get everything done.
2. I had a look at the MKT app and couldn't find RMX Donut Plains 1 in the favorite coverage (see image). Could you elaborate?
![IMG_3759](https://user-images.githubusercontent.com/79757836/195659038-0f9d5d97-6ef2-4c79-b4d7-d217e3e938e7.PNG)
3. Yes, I know 😓. I am currently in the process of rewriting the whole app because there are a few of those issues and all are caused by the current app structure. Although I might have an idea to get a temporary performance boost without much code change which I will try to implement with 1.2.0 as well.

**Reply by MusicIsTheBest** [reply]
Thanks and in response to #2:![image](https://user-images.githubusercontent.com/115706872/195726096-7ba3c87b-626f-45ac-853a-be1c8a7b9849.jpeg)

It's under favored (2 item course) but was given the Peach Cup tier upgrade this tour. Similar to Peach in the same cup this tour.

**Reply by TitusPosiadly** [reply]
Ah, yes. The coverage list you see when you open a card's details does not include the cup-specific boosts. This is because the card might not get a boost in another cup and would therefore show incorrect coverage. A good example of this is Cookie Land where the Yoshi Mii Suit gets a boost (top shelf) in the Diddy Kong Cup but doesn't (second shelf) in the Peach Cup. You probably already saw this but when you open the track in Tour -> Cups -> Peach Cup -> RMX Donut Plains 1 you will see that the tour page indeed includes the cup boosts. I hope that clears things up a bit.
